clap_derive/src/utils/mod.rs

```diff
@@ -11,3 +11,20 @@ pub use self::{
     spanned::Sp,
     ty::{inner_type, is_simple_ty, sub_type, subty_if_name, Ty},
 };
+
+use crate::item::Item;
+use syn::{Field, FieldsNamed};
+
+pub fn get_fields<'a>(
```

Comment:
`get` implies a lighter weight process than what is happening.  `parse` or `collect` might be more appropriate.

This should also convey this is specifically for `Args`.

So maybe `collect_args_fields`?